tests/nnapi/specs/V1_1/transpose_quant8_1.mod.py

```diff
@@ -1,6 +1,6 @@
 model = Model()
 i1 = Input("input", "TENSOR_QUANT8_ASYMM", "{2, 3, 4, 5}, 1.0, 0")
-perms = Parameter("perms", "TENSOR_INT32", "{4}", [2, 0, 1, 3])
+perms = Input("perms", "TENSOR_INT32", "{4}")
```

Comment:
Please do not modify this test file.

Reply:
Reverted the changes, with reasoning given in Issue #1875 . Thank you for the discussion.